Components/Decks/DraftDeckEditor.jsx

```diff
@@ -0,0 +1,428 @@
+                                type='text' onChange={ this.onChange.bind(this, 'deckName') } value={ this.state.deckName } />
+                            <Select name='faction' label='Faction' labelClass='col-sm-3' fieldClass='col-sm-9' options={ Object.values(this.props.factions) }
+                                onChange={ this.onFactionChange.bind(this) } value={ this.state.faction ? this.state.faction.value : undefined } />
+                            <Select name='agenda' label='Agenda' labelClass='col-sm-3' fieldClass='col-sm-9' options={ agendas }
+                                onChange={ this.onAgendaChange.bind(this) } value={ this.state.agenda ? this.state.agenda.code : undefined }
+                                valueKey='code' nameKey='label' blankOption={ { label: '- Select -', code: '' } } />
+
+                            { this.state.showBanners &&
+                                <div>
+                                    <Select name='banners' label='Banners' labelClass='col-sm-3' fieldClass='col-sm-9' options={ this.props.banners }
+                                        onChange={ this.onBannerChange.bind(this) } value={ this.state.selectedBanner ? this.state.selectedBanner.code : undefined }
+                                        valueKey='code' nameKey='label'
+                                        blankOption={ { label: '- Select -', code: '' } } button={ { text: 'Add', onClick: this.onAddBanner.bind(this) } } />
+                                    <div className='col-sm-9 col-sm-offset-3 banner-list'>
+                                        { banners }
+                                    </div>
+                                </div>
+                            }
+                        </form>
+                    </div>
+                    <div className='col-sm-1' />
+                    <div className='col-sm-6'>
+                        <DeckSummaryHeader
+                            deck={ this.getDeckFromState() }
+                            onCardMouseOut={ () => this.clearHoverCard() }
+                            onCardMouseOver={ card => this.updateHoverCard(card) } />
+                    </div>
+                </div>
+                <div className='row'>
+                    <div className='col-sm-12' style={ { 'text-align': 'center', 'margin-bottom': '1em' } }>
+                        Click cards under the "Remaining Drafted Cards" to move them into your deck and vice-versa.
+                    </div>
+                </div>
+                <div className='row'>
+                    <div className='col-sm-5'>
+                        <div className='form-group'>
+                            <label className='col-sm-12'>Remaining Drafted Cards:</label>
+                        </div>
+                        <div className='col-sm-12'>
+                            <CardTypeGroups
+                                cards={ this.state.remainingCards }
```

Comment:
this CardTypeGroups shows all remaining cards including agendas. Maybe we have to find some way to filter out agendas from the clickable cards as those are already in the agenda drop down
![grafik](https://user-images.githubusercontent.com/6191998/136658236-d49a186a-28ff-452b-82c8-0f2d8db972bc.png)